src/hu_nmt/data_augmentator/graph_mappers/graph_similarity_base.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+import networkx as nx
+from typing import Tuple, Dict
+
+Edge = Tuple[str, str, Dict]
+Node = str
+
+class GraphSimilarityBase(ABC):
+
+    @abstractmethod
+    def get_similarity_from_graphs(self, src_graph: nx.DiGraph, tgt_graph: nx.DiGraph) -> float:
+        raise NotImplementedError
+
+    def _get_root(self, graph: nx.DiGraph):
+        node = [n for n, d in graph.in_degree() if d == 0][0]
+        return node
```

Comment:
Why is this function in this class? I think it should belong in the graph wrapper. In fact, it's already there: https://github.com/attilanagy234/syntax-augmentation-nmt/blob/main/src/hu_nmt/data_augmentator/wrapper/dependency_graph_wrapper.py#L17